packages/engine-core/package.json

```diff
@@ -16,7 +16,7 @@
   "bugs": "https://github.com/prisma/prisma/issues",
   "devDependencies": {
     "@types/jest": "27.0.1",
-    "@types/node": "12.20.19",
+    "@types/node": "16.6.2",
```

Comment:
See https://github.com/prisma/prisma/pull/8842#issuecomment-903525303

Reply:
What do we think about this comment now?

Reply:
This is reserved for their experimental `fetch` API, which we don't use. This is type-only (to satisfy TS) and `Blob` isn't exported or used, so I consider this change fine.